Lib9c/Model/State/StateExtensions.cs

```diff
@@ -88,17 +88,13 @@ public static HashSet<T> ToHashSet<T>(this IValue serialized, Func<IValue, T> de
 
         #endregion
 
-        //FIXME (Text) 대신 (Integer) 로 직렬화해야함
```

Comment:
Is it okay that we don't care about this comment now?

Reply:
I think this project was already in a state where we didn't have to worry about this comment.
However, since this change required a chain reset, I was worried about when to do it.
So, I am looking forward to putting up PR first and merging them at the right time.